### DISCUSSION TITLE
Adding "Eligibility" as an Item attribute in the core spec

### DISCUSSION THREAD

**Discussion by SavitaMuley** [beckn/protocol-specifications#415] [Proposals]
As we implement multiple use cases such as Jobs, Financial Services, and Welfare, there is a need to introduce the notion of eligibility for an Item. In case of Financial Services or Welfare, there are specific pre-requisites that need to be fulfilled before customer can avail that resource - eligibility criteria could be based on illustrative categories as below
1. Demography e.g., age, gender, location for Welfare
2. Social Credentials and/ or Health determinants e.g., marital status, health condition for enabling healthcare services under Welfare programs
3. Academic details e.g., education qualifications or Scores/marks to assess Scholarships
4. Economic status such as Income, Occupation, Turnover, land-holding, capital asset holding to assess Credit or Insurance products fitment

Proposal is to introduce Item attribute that will enable NPs to enable evaluation of pre-requisites before Order is submitted. 

Eligibility attribute can have sub attributes to define categorization (illustrated above) for each ruleset. Depending on how we want to enable eligibility evaluation, there may be need to introduce Rules (or ability to use operators such as >, <, =, AND, OR, IN-LIST, etc. to construct rules)

**Comment by vbaheti** [comment]
This would be very useful in couple of other use cases where business on want to target niche (specific type of) customers for offering ultra premium services.
